pkg/components/internal/testutil/jsonpath.go

```diff
@@ -103,6 +104,29 @@ func MatchJSONPathInt64Value(t *testing.T, yamlConfig string, jsonPath string, e
 	}
 }
 
+// MatchJSONPathJSONValue is a helper function for component unit tests. It compares the JSON values
```

Comment:
Any new util functions that need not be visible should be part of the internal package 

example this file : `internal/util_test.go`

Reply:
But this `pkg/components/internal/testutil/jsonpath.go` has all the relevant code that deals with finding values at JSON path, converting into appropriate concrete data types and comparing values.

Reply:
And it will be needed in various `_test.go` files, I don't fully understand what are you suggesting?

Reply:
I'm suggesting to use the internal package for utility functions such as the one below

https://github.com/kinvolk/lokomotive/blob/master/internal/util_test.go